dark/assets/css/styles.css

```
@@ -1051,6 +1051,12 @@ html {
   z-index: -1;
 }
 
+#contact center {

```

Can you create a class as contact-text, then it wll be `#contact .contact-text` @Rishav1707 
```suggestion
#contact .contact-text {
```